static/index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Movies APP</title>
+    <link href="./../static/style.css" rel="stylesheet">
+  </head>
+  <body>
+    <main>
+      <h1 class="site__title">Movies APP</h1>
+
+      <header class="movie__wrapper">
+        <h2 class="movie__title">The Matrix</h1>
+        <div class="movie__description">
+        <p>"The Matrix," directed by the Wachowskis, is a 1999 science fiction film that explores themes of reality, freedom, and control. The story follows Thomas Anderson (Keanu Reeves), a computer hacker known by the alias "Neo." Neo is disillusioned with his mundane life and haunted by mysterious online encounters with a figure named Morpheus (Laurence Fishburne), who reveals a shocking truth: the world as Neo knows it is a simulated reality called the Matrix.</p>
+
+<p>Morpheus explains that in reality, intelligent machines have enslaved humanity, using their bodies as an energy source while keeping their minds imprisoned in the Matrix. Neo learns he is "The One," prophesied to liberate humanity from the machines. Morpheus and his crew, including the skilled fighter Trinity (Carrie-Anne Moss), train Neo to manipulate the Matrix's code, giving him superhuman abilities within the simulation.</p>
+
+<p>The group battles against the Matrix's sentient security programs, known as Agents, led by the relentless Agent Smith (Hugo Weaving). As Neo's powers grow, he confronts his destiny, culminating in a climactic showdown with Agent Smith. Neo ultimately embraces his role as humanity's savior, heralding the beginning of the fight to free humanity from the Matrix's grip. The film ends with Neo's promise to expose the truth and lead the resistance against the machines.</p></div>
+        <img class="movie__poster" src="./../assets/matrix.png" alt="IMDb Logo" />
+      </header>
+
+      <section>
+        <h2 class="site__title">Movie Reviews</h2>
+
+        <form class="review__form">
+          <textarea id="movie-review" name="movie-review" required placeholder="Add review"></textarea>
+
+          <div class="rating__labels">
+
+            <label class="rating__label">
+              <span class="rating__star" aria-hidden="true"></span>
+              <div class="rating__number">5</div>
+              <input class="rating__radio" type="radio" id="rating-5" name="rating" value="5" />
+            </label>
+
+            <label class="rating__label">
+              <span class="rating__star" aria-hidden="true"></span>
+              <div class="rating__number">4</div>
```

Comment:
What are these divs used for?

Reply:
`rating__number`? For accessibility

Reply:
But if the radio input visibility is handled differently then could be removed

Reply:
Wouldn't a label with a correct for attribute be better?

Reply:
That does nothing accessibility wise.
If you are referring the `label` relationship to the `input radio` field, the `radio` is inside the `label` making the `for` attribute redundant.

Reply:
That is what I was referring to. I missed that it is wrapped, it all makes sense now :)